gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoint-group/general/api-endpoint-group-general.component.html

```diff
@@ -0,0 +1,51 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form *ngIf="!!generalForm" [formGroup]="generalForm" autocomplete="off" gioFormFocusInvalid>
+  <mat-card>
+    <!-- Name -->
+    <div class="card__endpoint-group-name">
+      <div class="card__endpoint-group-name__label">Name</div>
```

Comment:
I feel like there is a style problem:

![image](https://github.com/gravitee-io/gravitee-api-management/assets/47851994/fabd511b-47e0-4086-b8a8-dd2d68acf867)

The name is very close to the input.

I wonder if you need that label to be honest, since the component already indicates that it is "Name" thanks to mat-label